Update hub-install.adoc (#366)
uyuni-project/uyuni-docs: modules/large-deployments/pages/hub-install.adoc

[[lsd-hub-install]]
= Hub Installation

Before you begin, you need to install the ``hub-xmlrpc-api`` package, and configure the Hub Server to use the API.

.Procedure: Installing and Configuring the XMLRPC API
. On the Hub Server, or on a host that has access to all peripheral servers' XMLRPC APIs, install the ``hub-xmlrpc-api`` package.
The package is available in the {productname} {productnumber} repositories.

> * If the ``hub-xmlrpc-api`` package is installed on the Hub then no further configuration is necessary. The daemon can be started via systemd as a ``hub-xmlrpc-api.service`` file is provided by the package.

Reviewer: ```suggestion
. If the ``hub-xmlrpc-api`` package is installed on the Hub then no further configuration is necessary and you can start the service using systemd.
The ``hub-xmlrpc-api.service`` file is provided by the package.
```

> * If the ``hub-xmlrpc-api`` package is installed on a separate host then verify if ``HUB_API_URL`` field defined in the configuration file available at ``/etc/hub/hub.conf`` is pointing to the correct Hub instance and modify it as required.

Reviewer: ```suggestion
Alternatively, if the ``hub-xmlrpc-api`` package is installed on a separate host, you need to verify that the configuration file is pointing to the correct Hub instance.
Open the [path]``/etc/hub/hub.conf`` configuration file and check that the ``HUB_API_URL`` field is correct.
```

. OPTIONAL: In the configuration file, check the settings for the timeout parameters ``HUB_CONNECT_TIMEOUT`` and ``HUB_REQUEST_TIMEOUT``.

> * ``HUB_CONNECT_TIMEOUT`` refers to the maximum number of seconds to wait for a response when connecting to a Server.

Reviewer: ```suggestion
* ``HUB_CONNECT_TIMEOUT`` refers to the maximum number of seconds to wait for a response when connecting to a server.
```

> * ``HUB_REQUEST_TIMEOUT`` refers to the maximum numbr of seconds to wait for a response when calling a Server method.

Reviewer: ```suggestion
* ``HUB_REQUEST_TIMEOUT`` refers to the maximum number of seconds to wait for a response when calling a server method.
```

> The default values for these parameters should suffice in most settings.

Reviewer: ```suggestion
In most cases, these parameters can be left at their default values.
```
